src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.ServiceManagement.psd1

```diff
@@ -0,0 +1,108 @@
+﻿#  
```

Comment:
Why is ServiceManagement necessary in this manifest name.  Consider a shorter name